src/components/featuredContent/featuredContent.tsx

```diff
@@ -0,0 +1,45 @@
+import {
+  FeaturedContent,
+  Button,
+  Heading,
+} from "@nypl/design-system-react-components";
+import React from "react";
+import featuredContentData from "@/data/featuredContentData";
+import { FeaturedContentData } from "@/types/FeaturedContentData";
+
+const FeaturedContentComponent = ({ testRandomNumber }) => {
+  const randomNumber =
+    testRandomNumber !== undefined
```

Comment:
You can update this to `testRandomNumber || Math.floor(Math.random() * 2);`. Minor comment but this works just the same.

Reply:
Okay and this drove me crazy bc it does appear to be the exact same thing, but with just the OR operator it fails to render the case where `testRandomNumber` is 0, because OR will take 0 as falsy. This makes me want to make the options 1 or 2 instead of 0 or 1, but it's indexing into an array, so that feels wrong. Open to ideas though

Reply:
Oh, that's right that we _want_ 0 as a value. That's my mistake. The original is fine.